rocm/ubi9-python-3.9/Dockerfile

```diff
@@ -18,6 +18,9 @@ WORKDIR /opt/app-root/bin
 ARG ROCM_VERSION=6.1
 ARG AMDGPU_VERSION=6.1
 
+# default: same targets and ROCm version as https://github.com/tiran/instructlab-containers/blob/main/containers/rocm/Containerfile.c9s#L47
+ARG AMDGPU_TARGETS=gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack-;gfx90a:xnack+;gfx942;gfx1030;gfx1100
```

Comment:
Can we get some official set of AMD GPUs we should/need to support? I mean - are we sure that this default is what we want?

Reply:
> Looks good, but we should really assure that this isn't missing in the final image for any use-case we should support.

It's much better to miss something and add it later, than trying to remove things afterwards when we actually have users, imo.

Reply:
Plan is to support: AMD MI210, MI300 and later on MI250
Based on the strat we have https://issues.redhat.com/browse/RHOAISTRAT-135.

i agree, we can add the details if the opportunity presents, and try this out for now.
however what are these gfx files, are we aware what functionality is this removing ?